src/store/courses/actionCreators.js

```diff
@@ -0,0 +1,26 @@
+import {
+	ADD_COURSE,
+	DELETE_COURSE,
+	SAVE_COURSES,
+	UPDATE_COURSE,
+} from './actionTypes';
+
+export const saveCourses = (courses) => ({
```

Comment:
Please rename this method because you do not save, you get)

Reply:
changed :)